config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get 'users/index'
+  get 'users/show'
+  get 'posts/index'
+  get 'posts/show'
```

Comment:
- [ ] I would recommend removing these paths since you have already mentioned the path for users and posts on line 7 to 9, so no need for this.

Reply:
Okay Thanks , I have removed it